spec.html

```diff
@@ -33480,23 +33480,22 @@ <h1>String.prototype</h1>
         <h1>String.raw ( _template_, ..._substitutions_ )</h1>
         <p>The `String.raw` function may be called with a variable number of arguments. The first argument is _template_ and the remainder of the arguments form the List _substitutions_. The following steps are taken:</p>
         <emu-alg>
-          1. Let _numberOfSubstitutions_ be the number of elements in _substitutions_.
+          1. Let _substitutionCount_ be the number of elements in _substitutions_.
           1. Let _cooked_ be ? ToObject(_template_).
-          1. Let _raw_ be ? ToObject(? Get(_cooked_, *"raw"*)).
-          1. Let _literalSegments_ be ? LengthOfArrayLike(_raw_).
-          1. If _literalSegments_ &le; 0, return the empty String.
-          1. Let _stringElements_ be a new empty List.
+          1. Let _literals_ be ? ToObject(? Get(_cooked_, *"raw"*)).
```

Comment:
Nit: I'm calling these "blocks" in the `String.dedent` proposal. Essentially, a template is 1+ blocks, and each block can have multiple lines.

Reply:
Yeah, there doesn't seem to be consistency here. [String.raw](https://tc39.es/ecma262/multipage/text-processing.html#sec-string.raw) refers to them as "literal segments", the [Template Literals grammar](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-template-literals) section arguably as "spans", [GetTemplateObject](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-gettemplateobject) and [TemplateStrings](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-static-semantics-templatestrings) and MDN as "[template] strings", etc.

Reply:
I'd be fine with calling the segments or spans. But "literal lines" (`String.dedent` is concerned with the individual lines) sounds very confusing.